src/main/java/BattleshipModel.java

```diff
@@ -58,6 +59,52 @@ public void fire(Coordinate shot){
 
     }
 
+    public void AIFire(Board target)
+    {
+        ArrayList<Coordinate> ships = target.getAllShips();
+
+
+        ArrayList<Coordinate> hits = target.getHits();
+        ArrayList<Coordinate> shots = target.getAllShots();
+        Coordinate previousHit= hits.get(hits.size() - 1);
+
+        int x = previousHit.getAcross();
+        int y = previousHit.getDown();
+
+        Coordinate up = new Coordinate(x, y - 1);
+        Coordinate down = new Coordinate(x, y + 1);
+        Coordinate left = new Coordinate(x - 1, y );
+        Coordinate right = new Coordinate(x + 1, y);
+
+        boolean checkUp = true;
+        boolean checkDown = true;
+        boolean checkLeft = true;
+        boolean checkRight = true;
+
+        for(int i = 0; i < shots.size() - 1; i++)
+        {
+            Coordinate curr = shots.get(i);
+            if(up.getAcross() == curr.getAcross() && up.getDown() == curr.getDown())
+            {
+                checkUp = false;
+            }
+            if(down.getAcross() == curr.getAcross() && down.getDown() == curr.getDown())
+            {
+                checkDown = false;
+            }
+            if(left.getAcross() == curr.getAcross() && left.getDown() == curr.getDown())
+            {
+                checkLeft = false;
+            }
+            if(right.getAcross() == curr.getAcross() && right.getDown() == curr.getDown())
+            {
+                checkRight = false;
+            }
+
+            }
+
```

Comment:
If none of the if statements are true, we will need to add an else statement to randomly fire on the opponents board. This will satisfy the easy AI requirement.